src/VisualStudio/Core/Def/Implementation/Options/RoamingVisualStudioProfileOptionPersister.cs

```diff
@@ -190,6 +177,24 @@ public bool TryFetch(OptionKey optionKey, out object value)
             return true;
         }
 
+        private bool DeserializeCodeStyleOption<T>(ref object value)
```

Comment:
Because the value is in/out.  i.e. we get it as a string, but we may write back out to is as a CodeStyleOption.